lib/RestructureCFG/InlineDispatcherSwitch.cpp

```diff
@@ -1002,8 +1046,11 @@ static RecursiveCoroutine<ASTNode *> simplifySwitchBreakImpl(ASTNode *Node) {
 /// `case`.
 ASTNode *simplifySwitchBreak(ASTTree &AST, ASTNode *RootNode) {
```

Comment:
Here too I think we don't need the `RootNode` argument. All the code assumes `RootNode` is the root of `AST`, so `AST` is enough.

Reply:
Done this in a dedicated commit b9a8a08f7f6160f57da4506aea7fce6f85550453

Reply:
:+1: